dfanalyzer/graph.py

```diff
@@ -0,0 +1,195 @@
+
+import dask.dataframe as dd
+import re
+import os
+import glob
+import pandas as pd
+
+class GrepIO:
+    def __init__(self, ddf=None, app_name = '',cp_dir='', existing = False):
```

Comment:
Add a class level comment to describe which the class is doing.
Please include the init parameters, their description and if relevant possible values.